solr/core/src/java/org/apache/solr/util/PackageTool.java

```diff
@@ -261,44 +261,44 @@ protected void runImpl(CommandLine cli) throws Exception {
         .argName("URL")
         .hasArg()
         .required(true)
-        .desc("Address of the Solr Web application, defaults to: " + SolrCLI.DEFAULT_SOLR_URL)
+        .desc("Address of the Solr Web application, defaults to: " + SolrCLI.DEFAULT_SOLR_URL + '.')
         .build(),
 
         Option.builder("collections")
         .argName("COLLECTIONS")
         .hasArg()
         .required(false)
-        .desc("List of collections. Run './solr package help' for more details.")
+        .desc("Specifies that this is to be a collection(s) level action.")
```

Comment:
I don't know if this is accurate, but maybe something like:

"Specifies that this action should affect packages for the given collections only, excluding cluster level packages."

And for the cluster one:

"Specifies that this action should affect cluster-level packages only."

Reply:
Love it!